person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, parent_permission: true, name: 'Unknown')
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
```

Comment:
- **[OPTIONAL]** It would be nice to move these attr_accessor and attr_reader to the top of the class.

Reply:
Alright!... :+1:  :rocket: